tests/test_struct.py

```diff
@@ -1,9 +1,11 @@
 """Automate struct tests."""
 import unittest
 
+from pyof.utils import unpack
 from tests.raw_dump import RawDump
 
 
+# to be removed and replace by TestMsgDumpFile
```

Comment:
I'm waiting for you guys to review this so I can apply the changes to v0x04 and remove the old class in this same PR